Examples/react-application-insights-demo/public/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
+  "icons": [
```

Comment:
what does the user need to install to get this to run? Do you need a package.json?

Reply:
Added a note in README on how the whole repo is just generated code from create-react-app except for App.jsx